tokio/src/io/driver/mod.rs

```diff
@@ -212,27 +215,6 @@ impl Driver {
     }
 }
 
-impl Drop for Driver {
-    fn drop(&mut self) {
-        (*self.inner.resources.lock()) = self.resources.take();
-    }
-}
-
-impl Drop for Inner {
```

Comment:
I think we still want this impl, to shutdown on drop?

Reply:
done